packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -14,13 +14,17 @@ flutter:
         pluginClass: FLTPathProviderPlugin
       macos:
         default_package: path_provider_macos
-
+      linux:
+        default_package: path_provider_linux
 
 dependencies:
   flutter:
     sdk: flutter
   path_provider_platform_interface: ^1.0.1
   path_provider_macos: ^0.0.4
+  path_provider_linux:
+    path: ../path_provider_linux
```

Comment:
This should not be using a path-based dependency. I think you'll need to publish path_provider_linux first, then endorse it in a follow-up.

Reply:
Yeah I know, I was just following the windows example link you sent me. I'll go ahead and revert the changes to the path_provider, and get the linux plugin in first in this pull request before endorsing it in another.